app/models/host/managed.rb

```diff
@@ -736,6 +771,11 @@ def clone
       host.compute_attributes = host.compute_resource.vm_compute_attributes_for(self.uuid)
     end
     host.refresh_global_status
+
+    host.host_facets.each do |facet|
```

Comment:
`puppet_status = 0` was moved to `refresh_global_status` so we don't *need* the facet to support this at the moment, let's wait until it's needed to add it